pkg/plugin/v3/api.go

```diff
@@ -43,6 +43,9 @@ import (
 // TODO: remove this when a better solution for using addons is implemented.
 const KbDeclarativePatternVersion = "v0.0.0-20200522144838-848d48e5b073"
 
+// AddonPlugin is the name of the plugin for addons.
+const AddonPlugin = "addon"
```

Comment:
The Addon still not a plugin. We have an issue raised to change that. See: https://github.com/kubernetes-sigs/kubebuilder/issues/1543

Reply:
Okay will change the comment. Thanks :)

Reply:
Done